INSTALL.md

```diff
@@ -73,14 +73,14 @@ To use gflags in a Bazel project, map it in as an external dependency by editing
 your WORKSPACE file:
 
     git_repository(
-        name = "gflags_git",
+        name = "com_googlesource_code_gflags",
```

Comment:
I don't care too much,  but RE2 follows this style of name, and RE2 is the only project that I could find that has an importable BUILD.